### DISCUSSION TITLE
Couldn't upgrade to 1.8.1 - SQL error

### DISCUSSION THREAD

**Discussion by jsanglier** [YOURLS/YOURLS#3031] [Q&A] [answered]
Hi, when I tried to upgrade from 1.7.9 to 1.8.1, I got this:

SQLSTATE[HY000]: General error: 1709 Index column size too large. The maximum column size is 767 bytes

I have reverted to my backed up 1.7.9 install for now, but how do I fix this?

I am suddenly out of my depth! haha

**Comment by dgw** [comment, marked as answer]
This has come up several times. See https://github.com/YOURLS/YOURLS/discussions/2857#discussioncomment-426530 and breadcrumbs leading from there.

**Reply by jsanglier** [reply]
Ta
